native/cocos/scene/Camera.cpp

```diff
@@ -424,6 +424,15 @@ void Camera::updateAspect(bool oriented) {
     _isProjDirty = true;
 }
 
+void Camera::setSystemWindowId(uint32_t windowId) {
```

Comment:
It is not a good idea to set system window id here. Camera should only set render window.